packages/block-library/src/query/edit/inspector-controls/format-controls.js

```diff
@@ -68,7 +68,7 @@ export default function FormatControls( { onChange, query: { format } } ) {
 		.filter( Boolean );
 
 	const suggestions = formats
-		.filter( ( item ) => ! format.includes( item.value ) )
+		.filter( ( item ) => ! normalizedFormats.includes( item.value ) )
```

Comment:
Just to refresh my memory, this filter removes values from the suggestions already assigned to the Query, correct?

Reply:
Yes.
Shall I add a comment about it? I still need to create a new PR to apply the changes requested in the review of the sync with core, I could add it then.